tests/goth/test_run_blender.py

```diff
@@ -103,7 +103,7 @@ async def test_run_blender(
         requestor = runner.get_probes(probe_type=RequestorProbe)[0]
 
         async with requestor.run_command_on_host(
-            f"node {blender_path} --subnet-tag goth",
+            f"npm run --prefix {examples_dir} ts:blender -- -d --subnet-tag goth",
```

Comment:
just out of curiosity - what's the difference between these two invocations?

Reply:
in the first one we use node directly so we can only execute js file. we can't run ts file (typescript) in node engine. First we need to compile ts -> js. To avoid this in this test example (i wanted to test ts version in goth), we can use the ts-node tool, which compiles and runs the ts file on the fly. It is hidden under the `ts:blender` command in package.json, so if we want to run more advanced scripts than node {js_file} we need to define it in package.json and use npm, something like poetry and poe tool..